### PR TITLE
Update metadata.txt to include a changelog for 1.1.0

### PR DESCRIPTION
**Pull request by havatv**
Update metadata.txt to include changelog for 1.1.0

### CODE REVIEW

**Comment by mkondratyev85** [comment]
Current version of raster tracer is 0.1.1. I believe next release could be v.0.2, not 1.1.0 as in your proposal.

File: metadata.txt [hunk 1]
```diff
@@ -20,7 +20,10 @@ repository=https://github.com/mkondratyev85/raster_tracer/
 
 hasProcessingProvider=no
 # Uncomment the following line and add your changelog:
-# changelog=
+changelog=1.1.0
```
Comment (havatv) [inline metadata.txt:23]:
```suggestion
changelog=0.2.0
```

**Comment by havatv** [comment]
> Current version of raster tracer is 0.1.1. I believe next release could be v.0.2, not 1.1.0 as in your proposal.

Sorry about that. I only added the changelog, and for some reason I used 1.1.0 (probably a cut and paste from another plugin changelog)
I have now changed *version* and updated *changelog* in `metadata.txt`.

**Comment by mkondratyev85** [comment]
I've added new tag v0.2 and uploaded zip to QGIS plugins repository https://plugins.qgis.org/plugins/raster_tracer/ and now it waits for approval.

**Pull request merged**